src/main/java/de/slimecloud/slimeball/features/highlights/HighlightListener.java

```diff
@@ -0,0 +1,60 @@
+package de.slimecloud.slimeball.features.highlights;
+
+import de.cyklon.jevent.EventHandler;
+import de.cyklon.jevent.Listener;
+import de.slimecloud.slimeball.features.highlights.event.HighlightTriggeredEvent;
+import de.slimecloud.slimeball.main.SlimeBot;
+import lombok.RequiredArgsConstructor;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.*;
+import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@Listener
+@RequiredArgsConstructor
+public class HighlightListener extends ListenerAdapter {
+
+	private final SlimeBot bot;
+
+	@Override
+	public void onMessageReceived(@NotNull MessageReceivedEvent event) {
+		if (!event.isFromGuild()) return;
+
```

Comment:
The highlight shouldn't trigger for bots, or at least not for the bot sending the confirmation message

Reply:
I asked this question in tec chat, but have not received an answer to date. 😐